cmsl1t/plotting/efficiency.py

```diff
@@ -5,34 +5,48 @@
 import cmsl1t.hist.binning as bn
 from cmsl1t.utils.draw import draw, label_canvas
 from cmsl1t.utils.fit_efficiency import fit_efficiency
-from cmsl1t.io import to_root
+
 
 from rootpy.plotting import Legend, HistStack, Efficiency
 from rootpy.context import preserve_current_style
 from rootpy import asrootpy, ROOT
 
 
+# Hack to fix Efficiency.__iadd__ for now
```

Comment:
I opened an issue for this on rootpy which has now been fixed: 
https://github.com/rootpy/rootpy/issues/762
Will be available to us once version 0.9.2 is put on PyPi I guess.